plugins/SitesManager/lang/de.json

```diff
@@ -157,6 +157,66 @@
         "WebsitesManagement": "Webseitenverwaltung",
         "XManagement": "%s verwalten",
         "YouCurrentlyHaveAccessToNWebsites": "Sie haben derzeit Zugriff auf %s Webseiten.",
-        "YourCurrentIpAddressIs": "Ihre aktuelle IP-Adresse ist %s"
+        "YourCurrentIpAddressIs": "Ihre aktuelle IP-Adresse ist %s",
+        "SiteWithoutDataStartTrackingDataHeader": "Datenverfolgung starten",
+        "SiteWithoutDataStartTrackingDataDescriptionLine1": "Wählen Sie unten Ihre bevorzugte Tracking-Methode aus. Wenn Sie Hilfe benötigen, können Sie diese %1$sAnweisungen per E-Mail teilen%2$s oder %3$sweitere Benutzer einladen%4$s, Ihnen bei der Einrichtung zu helfen.",
+        "SiteWithoutDataStartTrackingDataDescriptionLine2": "Hinweis: Dieser Bildschirm verschwindet, sobald Daten für diese Website erfasst werden. Alternativ können Sie %1$shier klicken, um diese Seite für eine Stunde auszublenden%2$s.",
+        "SiteWithoutDataVue": "Vue.js",
```

Comment:
Generally there is no sense in having a fixed name only in a translations. It would be the same across all languages, so be better to remove that again (even if you hadn't added it)

Reply:
I'm happy to do that in a separate PR after having a chat since there is already an established pattern of links within the no-data screens for GTM, React etc and this would be an exception to that. So even though I support the removal it would take the scope of this PR too far. Is that ok that way?

Reply:
As they had been in the english translation file before it's fine to do it that way.